doc/decisions/operation_sequences.md

```diff
@@ -0,0 +1,253 @@
+# Operation sequences
```

Comment:
Something went wrong with the rebase.

Reply:
Looks like it, the rebase skipped my last commit (add broken links to whitelist) because that was already done in PR #4635 and a following `git pull` messed up the branch quite a bit.

Now, it hopefully should be ok.